<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Push to Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_connection">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connection Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Move to Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="velInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_position">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Position (50-950):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocity">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Velocity (100-1000):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected.</string>
    </property>
   </widget>
   <widget class="QLabel" name="posStatus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velStatus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
